### PR TITLE
fix: parse tx to address in activity

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-1483
Figma link (if any):

## What changed (plus any additional context for devs)

we're now showing the correct address

## What to test

check a send tx and see if address is the actual recipient and not the contract address, for tokens and nfts

### CODE REVIEW

File: lavamoat/build-webpack/policy.json [hunk 1]
```diff
@@ -1156,8 +1156,13 @@
...
-        "jest>@jest/core>jest-snapshot>@babel/traverse>@babel/generator>@jridgewell/trace-mapping>@jridgewell/sourcemap-codec": true,
-        "webpack>terser-webpack-plugin>@jridgewell/trace-mapping>@jridgewell/resolve-uri": true
+        "jest>@jest/core>jest-snapshot>@babel/traverse>@babel/generator>@jridgewell/trace-mapping>@jridgewell/resolve-uri": true,
+        "jest>@jest/core>jest-snapshot>@babel/traverse>@babel/generator>@jridgewell/trace-mapping>@jridgewell/sourcemap-codec": true
+      }
+    },
+    "jest>@jest/core>jest-snapshot>@babel/traverse>@babel/generator>@jridgewell/trace-mapping>@jridgewell/resolve-uri": {
+      "globals": {
+        "define": true
       }
```
Comment (magiziz) [inline lavamoat/build-webpack/policy.json:1166]: Was this change needed ?

**Review by magiziz — APPROVED** [review]
Looks good to me, just had one question.

File: src/core/utils/transactions.ts [hunk 2]
```diff
@@ -173,6 +173,19 @@ const getAssetFromChanges = (
...
+    direction: TransactionDirection;
+    asset: ParsedUserAsset;
+    address_to: Address;
+  }[],
+  type: TransactionType,
+) => {
+  if (type === 'sale')
+    return changes?.find((c) => c?.direction === 'out')?.address_to;
+  return changes[0]?.address_to;
+};
```
Comment (DanielSinclair) [inline src/core/utils/transactions.ts:187]: Found one edge case with an Approval where the user address is still displayed

**Review by DanielSinclair — APPROVED** [review]

File: src/core/utils/transactions.ts (hunk 2 above)
Comment (BrodyHughes) [reply src/core/utils/transactions.ts:187]: i am not seeing a to field at all on older approvals

**Review by BrodyHughes — APPROVED** [review]
lgtm
